packages/grafana-ui/src/themes/GlobalStyles/fonts.ts

```diff
@@ -4,7 +4,9 @@ import { GrafanaTheme2 } from '@grafana/data';
 
 export function getFontStyles(theme: GrafanaTheme2) {
   const grafanaPublicPath = typeof window !== 'undefined' && window.__grafana_public_path__;
-  const fontRoot = grafanaPublicPath ? `${grafanaPublicPath}fonts/` : 'public/fonts/';
+  const relativefontRoot = grafanaPublicPath ? `${grafanaPublicPath}fonts/` : 'public/fonts/';
+  // NI fork: use absolute URLs to avoid incorrect resolution in Firefox due to interaction between iframes and Emotion Global styles.
+  const fontRoot = `${window.location.origin}${window.document.querySelector('base')?.href ?? '/'}${relativefontRoot}`;
```

Comment:
Can we use the resolved base uri instead of trying to interpret the element configuration, i.e. [`document.baseURI`](https://developer.mozilla.org/en-US/docs/Web/API/Node/baseURI)

```ts
  const fontRoot = `${document.baseURI}${relativefontRoot}`;
```

Reply:
Yes, much better.

Reply:
Moot now